chain/client/src/client.rs

```diff
@@ -914,6 +933,7 @@ impl Client {
             protocol_version,
         )?;
 
+        span.record("chunk_hash", format!("{:?}", encoded_chunk.chunk_hash()));
```

Comment:
Avoid using `format` here. It will force formatting of the value regardless of whether the span is being emit or not. Use https://docs.rs/tracing/latest/tracing/field/fn.debug.html instead.

Reply:
This is a very nice addition to this span btw, I didn't know you could do this!

Reply:
done.